docs/UserGuide.md

```diff
@@ -99,6 +99,10 @@ Adds a person to the address book.
 
 Format: `add n/NAME (p/PHONE_NUMBER | e/EMAIL) [r/MODULECODE[-ROLETYPE]]+ [a/ADDRESS] [t/TAG]+ [d/DESCRIPTION]`
 
+<box type="important" seamless>
+This is the only place where the symbol differs from its intended use. The command accepts either one phone number, one email, or both.
+</box>
+
```

Comment:
actually regarding this now that i think about it, this only appears in the UG but in the app the user won't be able to discover this, is this ok